sdk/core/core-auth/package.json

```diff
@@ -67,7 +67,6 @@
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
     "@azure/core-tracing": "1.0.0-preview.9",
-    "@opentelemetry/api": "^0.10.2",
```

Comment:
@xirzec Just a sanity check, but we shouldn't be depending on `@opentelemetry/api` in our packages (except core-tracing), right?

Reply:
We gotta, since you can't configured known exports in rollup except for direct dependencies. 

If you remove this, rollup will die because it can't figure out that "@opentelemetry/api" in its config refers to a transitive dependency. @bterlson tried to fix this in rollup, but ultimately it got too messy for an easy fix.

Reply:
Oh interesting, alright, added it back :cry: